src/server/scripts/Events/fireworks_show.cpp

```diff
@@ -0,0 +1,903 @@
+/*
+* Copyright (C) 2008-2016 TrinityCore <http://www.trinitycore.org/>
```

Comment:
is this solely for 3.3.5a branch? 
For some reason I am getting a few G3D::Quat errors 

> src\server\scripts\Events\fireworks_show.cpp(861): error C2027: use of undefined type 'G3D::Quat'

> dep\g3dlite\include\G3D/Matrix3.h(97): note: see declaration of 'G3D::Quat'

> use of undefined type 'G3D::Quat'
and

 

> WorldObject::SummonGameObject': function does not take 3 arguments 

on the master branch when trying to patch this  for testing
It may be an error on my side.. But I wanted to check and make sure.
For the life of me I fail to see what I am missing